dev/PowerNotifications/PowerNotifications.cpp

```diff
@@ -0,0 +1,240 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <pch.h>
+#include <PowerNotifications.h>
+#include <PowerNotificationsPal.h>
+#include <PowerManager.g.cpp>
+
+namespace winrt::Microsoft::ProjectReunion::implementation
+{
+    auto factory = make_self<factory_implementation::PowerManager>();
```

Comment:
Making the factory as part of a static init could have issues. This make_self would run under the loader lock as part of loading the DLL. Some of the stuff under make_factory, like activating CoreApplication to cache the factory in the static lifetime store (and potentially initializing COM via CoIncrementMTAUsage if it's not already initialized) may not get along well with the loader lock.